test/unitTests/observers/docsLoggerObserver.test.ts

```diff
@@ -42,17 +42,17 @@ describe('DocsLoggerObserver', () => {
             let expectedOutput = `Failed to sign-in to Docs Build: Faked error msg\n\n`;
             assert.equal(loggerText, expectedOutput);
         });
-    });
-
-    it(`CredentialRetrieveFromLocalCredentialManager`, () => {
-        let event = new CredentialRetrievedFromLocalCredentialManager(fakedCredential);
-        observer.eventHandler(event);
 
-        let expectedOutput = `Successfully retrieved user credential from Local Credential Manager:\n`
-            + `    - GitHub Account: Faked User\n`
-            + `    - User email    : fake@microsoft.com\n`
-            + `\n`;
-        assert.equal(loggerText, expectedOutput);
+        it(`CredentialRetrieveFromLocalCredentialManager`, () => {
```

Comment:
update test to use `UserSignInSucceeded`

Reply:
it just a name, and it tests the different case with the `UserSignInSucceeded`

---
In reply to: [384403609](https://github.com/docascode/vscode-docs-build/pull/70#discussion_r384403609) [](ancestors = 384403609)